hack/arktos-up.sh

```diff
@@ -527,6 +527,12 @@ fi
 ${KUBECTL} --kubeconfig="${CERT_DIR}/admin.kubeconfig" apply -f "${KUBE_ROOT}/pkg/controller/artifacts/crd-network.yaml"
 # refresh the resource discovery cache after the CRD is created
 ${KUBECTL} --kubeconfig="${CERT_DIR}/admin.kubeconfig" api-resources &>/dev/null
+# Applying mizar cni
+if [[ "${CNIPLUGIN}" = "mizar" ]]; then
+  ${KUBECTL} --kubeconfig="${CERT_DIR}/admin.kubeconfig" apply -f https://raw.githubusercontent.com/CentaurusInfra/mizar/dev-next/etc/deploy/deploy.mizar.yaml
+  sudo systemctl restart containerd.service
```

Comment:
Why do we need to restart this after deploying yaml?

Reply:
When we deploy mizar, we use customized [containerd for Arktos](https://github.com/CentaurusInfra/containerd/releases/tag/tenant-cni-args).
If we do not restart the containerd, then arktos deployment stuck with ```Waiting for node ready at api server``` message.
We need to restart containerd so that it can recognize the CNI conf files.

Reply:
Containerd is a runtime. Today when we deploy yaml in vanilla kubernetes, we don't restart docker or whatever runtime that k8s is using. So I'm wondering why we need to do it here. Do we need to restart containerd with other CLI drivers in arktos today?

@yb01 Do you know if we need to restart containerd when using with flannel or other drivers?

Reply:
As with bridge, we are not required to restart containerd.
We are debugging this and will try to remove the containerd restart dependency if possible.